core-java-modules/core-java-reflection-2/src/test/java/com/baeldung/reflection/access/privatemethods/InvokePrivateMethodsUnitTest.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.reflection.access.privatemethods;
+
+import com.google.common.primitives.Longs;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Method;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class InvokePrivateMethodsUnitTest {
```

Comment:
JUnit 5 doesn't need the class to be `public`

Reply:
I made an assumption we were using JUnit 4 without checking the dependencies.  Oops.  I will keep this in mind for future updates.